Symfony/CS/Fixer/Symfony/SilencedDeprecationErrorFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+final class SilencedDeprecationErrorFixer extends AbstractFixer
+{
+    public function getDescription()
+    {
+        return 'Ensures deprecation notices are silenced.';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+        for ($index = $tokens->count() - 1; $index >= 0; --$index) {
+            $token = $tokens[$index];
+            if (!$token->isGivenKind(T_STRING) || 'trigger_error' !== $token->getContent()) {
+                continue;
+            }
+
+            $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $tokens->getNextTokenOfKind($index, array(T_STRING, '(')));
+            if (
+                'E_USER_DEPRECATED' === $tokens[$tokens->getPrevMeaningfulToken($end)]->getContent()
```

Comment:
Just for any weird case, we could also check against the constant value (`16384`).